packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -108,6 +108,7 @@ function readContext<T>(context: ReactContext<T>): T {
 
 function useContext<T>(context: ReactContext<T>): T {
   hookLog.push({
+    displayName: context.displayName,
     primitive: 'Context',
```

Comment:
Can't just 
```suggestion
    primitive: context.displayName || 'Context',
```
since the value of `primitive` is used to determine whether the hook should have an index i.e. we need another field to ensure anonymous and named contexts both have no hook index.